### PR TITLE
[wip]: New Themes (Sparkbox Feature Branch)

### PR DESCRIPTION
**Pull request by bryanbraun**
# Summary
This is a feature branch containing the theme work that Sparkbox is working on.

It exists to allow us to make good progress on the theming without placing undue review burden on Mike while he's less-available this month. For more details, see our Slack conversation from Dec 9th.

All commits in this branch have been code-reviewed by a Sparkbox developer. We expect that this branch can be checked and merged into the feature branch by SWS when Mike returns. 

# Need Review By (Date)
01/06/2020

# Urgency
low

# Steps to Test
1. 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr436-ona5aac8ikkxyl7sy2icb4u1puygitxy.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5df7c0a64f4ecf2931e2f879

Visual Diffs:
- https://dashboard.tugboat.qa/5df7c0a64f4ecf2931e2f879/visualdiffs

File: docroot/themes/humsci/humsci_basic/patterns/accordion/accordion.html.twig [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{% if variant == 'open' %}
```
Comment (sherakama) [inline docroot/themes/humsci/humsci_basic/patterns/accordion/accordion.html.twig:1]: Work like this is great for base themes. Would there be any objections if we (SWS) copy some of it into stanford_basic?

**Comment by pookmish** [comment]
So far this looks good. no concerns from me at this time. Appears to be just alot of copy pasta. which is understandable to integrate the new version of basic

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr436-ona5aac8ikkxyl7sy2icb4u1puygitxy.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5df7c0a64f4ecf2931e2f879

Visual Diffs:
- https://dashboard.tugboat.qa/5df7c0a64f4ecf2931e2f879/visualdiffs

**Comment by silverli** [comment]
going to close this for a moment to see if tugboat is better

**Pull request closed without merging**